release_notes/370-release-notes.adoc

```diff
@@ -0,0 +1,185 @@
+:_content-type: ASSEMBLY
+[id="release-notes-370"]
+= Red Hat Advanced Cluster Security for Kubernetes 3.70
+include::modules/common-attributes.adoc[]
+:context: release-notes-370
+
+toc::[]
+
+*Release date:* 23 May 2022
+
+{product-title} is an enterprise-ready, Kubernetes-native container security solution that protects your vital applications across build, deploy, and runtime. It deploys in your infrastructure and integrates with your DevOps tooling and workflows to deliver better security and compliance and to enable DevOps and InfoSec teams to operationalize security.
+
+[id="about-this-release-370"]
+
+{product-title} ({product-title-short}) 3.70 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+[id="new-features-370"]
+== New features and enhancements
+
+[id="verify-image-signatures-rn"]
+=== Verifying image signatures against Cosign public keys
+
+You can use {product-title-short} to ensure the integrity of the container images in your clusters by verifying image signatures against pre-configured keys. You can also create policies to block unsigned images and images that do not have a verified signature and enforce the policy by using an admission controller to stop unauthorized deployment creation. Cosign key signature verification is supported. See _Verifying image signatures_ in the {product-title-short} Operating documentation for more information.
+
+[id="integration-ecs-rn"]
+=== Autogeneration of Amazon ECR registry integrations
+
+Registry integrations for Amazon Elastic Container Registry (ECR) are now auto-generated for Amazon Web Services (AWS) clusters. This feature requires that the nodes' Instance Identity and Access Management (IAM) Role has been granted access to ECR. You can turn off this feature by disabling the EC2 instance metadata service in your nodes. See _Amazon ECR integrations_ in the {product-title-short} Integrating documentation for more information.
+
+[id="identify-missing-policies-rn"]
+=== Identifying missing Kubernetes network policies
+
+Kubernetes network policies are vital in helping to enable zero-trust networking within a cluster. They reduce the impact of network attacks by limiting the opportunity for lateral movement. By default, Kubernetes resources are not isolated. Applying network policies is a recommended best practice left to the end user.
+
+{product-title-short} 3.70 ships with a new default policy that allows you to easily identify deployments that are not restricted by any ingress network policy and to trigger violation alerts accordingly.
+
+- The default policy is named *Deployments should have at least one ingress Network Policy*. It is disabled by default.
+
+- This default policy uses a new policy criterion called *Alert on missing ingress Network Policy*.
+
+- To identify pod isolation gaps, you can clone this default policy or create a new one by using the policy criterion and enabling it on selected resources.
+
+[id="spring-vulnerabilities"]
```

Comment:
(not a problem, but the convention on other features seems to have these end in `-rn`)